client/post-editor/controller.js

```diff
@@ -276,7 +260,30 @@ export default {
 			unsubscribe = context.store.subscribe( startEditingOnSiteSelected );
 		}
 
-		renderEditor( context );
+		let analyticsPath, analyticsTitle;
+		switch ( postType ) {
+			case 'post':
+				analyticsPath = '/post/:site';
+				analyticsTitle = 'Post > ';
+				break;
+			case 'page':
+				analyticsPath = '/page/:site';
+				analyticsTitle = 'Page > ';
+				break;
+			default:
+				analyticsPath = `/edit/${ postType }/:site`;
+				analyticsTitle = 'Custom Post Type > ';
+		}
+		if ( postToCopyId ) {
+			analyticsTitle += 'New';
+		} else if ( postID ) {
+			analyticsPath += '/:post_id';
+			analyticsTitle += 'Edit';
+		} else {
+			analyticsTitle += 'New';
+		}
```

Comment:
This is still my personal nit, but I quite dislike building strings :D I find it very difficult to search for usages in the future.

Perhaps consider the following.
```jsx
const getPostPathAndTitle = function ( postType, postId, postToCopyId ) {
    const isPost = postType === 'post';
    const isPage = postType === 'page';
    if ( isPost && ( postToCopyId || ! postId ) ) {
            return '[ '/post/:site', 'Post > New' ]';
    }

    if ( isPost && postID ) {
        return '[ '/post/:site/:post_id', 'Post > Edit' ]';
    }

    if ( isPage && ( postToCopyId || ! postId ) ) {
            return '[ '/page/:site', 'Page > New' ]';
    }

    if ( isPage && postID ) {
        return '[ '/page/:site/:post_id', 'Page > Edit' ]';
    }

    if ( postToCopyId || ! postId ) {
            return '[ '/edit/:site', 'Custom Post Type > New' ]';
    }

    if ( postId ) {
        return '[ '/edit/:site/:post_id', 'Custom Post Type > Edit' ]';
    }
}
// in render
const [ analyticsPath, analyticsTitle ] = getPostPathAndTitle( postType, postId, postToCopyId );
```